version.php

```diff
@@ -26,7 +26,7 @@
 
 defined('MOODLE_INTERNAL') || die;
 
-$plugin->version = 2019101001;
+$plugin->version = 2020072003;
```

Comment:
You have to restore the version to what we have 2019101001

Reply:
This is fixed now